### ISSUE TITLE
Accessibility - Unable to open selected file from the files explorer with voiceover [macOS]

### ISSUE THREAD

**Issue by adincebic** [Microsoft/vscode#57066]
- VSCode Version: 1.26.1
- OS Version: macOS High Siera 10.13.6

Steps to Reproduce:

1. Turn voiceover on by pressing command plus F5
2. Launch the VSCode
3. Press command plus shift plus E a couple of times until you hear "Files explorer table"
4. Now, press ctrl plus option plus down arrow to interact with the files explorer.
5. Try navigating with up or down arrow.
NOTE: If you disable quick navigation (press left and right arrow at the same time) and you may be able to navigate between files.

Expected result:
By interacting with the file navigator, I should be able to use arrows to navigate between files regardless of whether quick navigation is on or off. Also, I should be able to open selected file by pressing ctrl plus shift plus space.

Actual result:
When the quick navigation is on, voiceover only pronounces "Empty table". When quick navigation is off, voiceover reads the filenames but I am not able to open them.

What I tried:
I tried pressing the enter key when file is selected, but it only tries to rename the file.

Reference suggestion:
You can try using XCode navigator with voiceover as a comparison.

Does this issue occur when all extensions are disabled?: Yes/No
Yes

**Comment by isidorn** [comment]
Cmd + down should open the file just like the Finder on mac.
You can also customise this keybinding by changing the list.select keybinding in Preferences > Keyboard Shortcuts.

Does this solve your issue?

**Comment by adincebic** [comment]
Oh yes, that works fine.

**Comment by isidorn** [comment]
Great to hear.
